src/routes/user.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const userRouter = express.Router();
+const { getUsers } = require('../controllers/user');
+
+// Define a route to display users to admin
+userRouter.get('/api/users', getUsers);
```

Comment:
```suggestion
userRouter.get('/api/users', isAuth, checkIfAdmin, getUsers);
```

the user must be authenticated and admin to be able to fetch the users list